packages/format-currency/test/get-cached-formatter.ts

```diff
@@ -0,0 +1,39 @@
+import { getCachedFormatter } from '../src/get-cached-formatter';
+
+describe( 'getFormatter', () => {
```

Comment:
Ah! Thank you for adding tests for the caching. That's the one thing I had never automated tests for.

Reply:
We also had some benchmark tests in https://github.com/Automattic/wp-calypso/pull/99110 if you ever want to use/ refer to